planseqlearn/psl/robosuite_mp_env.py

```diff
@@ -917,12 +1242,12 @@ def set_object_pose(self, object_pos, object_quat, obj_idx=0):
             self.sim.data.qpos[self.hinge_qpos_addr] = object_pos
             self.sim.data.qpos[self.handle_qpos_addr] = object_quat
         elif "NutAssembly" in self.env_name:
-            if self.env_name.endswith("Square"):
+            if "gold" in obj_name:
                 nut = self.nuts[0]
-            elif self.env_name.endswith("Round"):
+            elif "silver" in obj_name:
                 nut = self.nuts[1]
-            elif self.env_name.endswith("NutAssembly"):
-                nut = self.nuts[1 - obj_idx]  # first nut is round, second nut is square
+            # elif self.env_name.endswith("NutAssembly"):
+            #     nut = self.nuts[1 - obj_idx]  # first nut is round, second nut is square
```

Comment:
why was this commented? just making sure its not a bug

Reply:
i removed this because 1) no longer using obj_idx and 2) the gold nut being nut 0 and silver nut being nut 1 is true across all environments